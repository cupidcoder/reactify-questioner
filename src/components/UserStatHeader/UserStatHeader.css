.UserStatHeader {
  display: flex;
  justify-content: space-between;
  width: 100%;
  padding: 2rem;
  border-bottom: 1px solid rgba(46,62,72,.12);
}

.UserStatHeader h3:first-child {
  font-size: 1.5rem;
}

.Joined {
  font-style: italic;
  font-size: 1.2rem;
  font-weight: 700;
}
