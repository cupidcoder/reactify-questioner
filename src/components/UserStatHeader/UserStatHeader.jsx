import React from 'react';
import style from './UserStatHeader.css';

const UserStatHeader = ({ name, joined }) => (
  <div className={style.UserStatHeader}>
    <h3>{name}</h3>
    <h3>
      <span className={style.Joined}>Joined:</span>
      {' '}
      {joined}
    </h3>
  </div>
);

export default UserStatHeader;
