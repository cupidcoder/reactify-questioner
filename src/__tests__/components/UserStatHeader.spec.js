import React from 'react';
import { shallow } from 'enzyme';
import UserStatHeader from '../../components/UserStatHeader/UserStatHeader';

describe('UserStatHeader', () => {
  it('Should correctly save snapshot of the UserStatHeader', () => {
    const wrapper = shallow(<UserStatHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
