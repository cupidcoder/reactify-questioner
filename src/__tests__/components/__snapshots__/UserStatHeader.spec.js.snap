// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserStatHeader Should correctly save snapshot of the UserStatHeader 1`] = `ShallowWrapper {}`;
